tasks/install.yml

```diff
@@ -1,16 +1,12 @@
 ---
-- name: "Add apt key"
-  apt_key:
-    url: https://repos.influxdata.com/influxdb.key
-    state: present
 
-- name: "Add apt repo"
-  apt_repository:
-    repo: "deb https://repos.influxdata.com/{{ ansible_facts['distribution']|lower }} {{ ansible_facts['distribution_release'] }} stable"
-    state: present
-    filename: influxdb
+- name: Prepare Telegraf installation
+  shell: |
```

Comment:
Let's add `creates: /etc/apt/sources.list.d/influxdata.list` to ensure that this task runs only once on server, I think it's good practice https://docs.ansible.com/ansible/latest/collections/ansible/builtin/shell_module.html#parameter-creates